LuaMenu/widgets/gui_battle_room_window.lua

```diff
@@ -1314,9 +1314,11 @@ local function SetupVotePanel(votePanel, battle, battleID)
 	local function SetMultiPollCandidates(candidates)
 		for i = 1, MULTI_POLL_MAX do
 			if candidates[i] then
+				-- displayName is the map's internal name and its dimensions in a human friendly way
+				local mapName = candidates[i].displayName or candidates[i].name
```

Comment:
This is a very defensive null check in case this function is called from somewhere else I didn't account for (otherwise I could put the fallback in `interface_zerok`)